photo-app/db/migrate/20220113123441_devise_create_users.rb

```diff
@@ -15,11 +15,11 @@ def change
       t.datetime :remember_created_at
 
       ## Trackable
-      # t.integer  :sign_in_count, default: 0, null: false
-      # t.datetime :current_sign_in_at
-      # t.datetime :last_sign_in_at
-      # t.string   :current_sign_in_ip
-      # t.string   :last_sign_in_ip
+      t.integer  :sign_in_count, default: 0, null: false
```

Comment:
Did you revert this migration in heroku?

Reply:
ya i ran revert then migrate again.